src/Elastic.Apm/Libraries/Ben.Demystifier/EnhancedStackFrame.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Ben A Adams. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+#nullable enable
+namespace System.Diagnostics
+{
+	public class EnhancedStackFrame : StackFrame
```

Comment:
Does this need to be public? Ditto for any other public types

Reply:
No - I made everything internal.